### ISSUE TITLE
[BUG] Custom CSS is no longer recognized.

### ISSUE THREAD

**Issue by josephgarnier** [shd101wyy/vscode-markdown-preview-enhanced#1812]
**Describe the bug**
**Since the last update**, the custom CSS style is no longer recognized, even in using the "Developer tools".

**Environment**
- OS: Linux
- VSCode: 1.83.0
- Markdown Preview Enhanced: 0.8.1

**To Reproduce**
1. Use a [custom CSS](https://shd101wyy.github.io/markdown-preview-enhanced/#/customize-css)

This CSS code worked fine before the update :

```css

/* Please visit the URL below for more information: */
/*   https://shd101wyy.github.io/markdown-preview-enhanced/#/customize-css */

.markdown-preview.markdown-preview {
  // modify your style here
  // eg: background-color: blue;
	
	font-family: var(--markdown-font-family, -apple-system, BlinkMacSystemFont, "Segoe WPC", "Segoe UI", system-ui, "Ubuntu", "Droid Sans", sans-serif);
	font-size: var(--markdown-font-size, 14px);
	padding: 0 26px;
	line-height: var(--markdown-line-height, 22px);
	word-wrap: break-word;
	
	padding-top: 1em;

	/* Reset margin top for elements */
	h1, h2, h3, h4, h5, h6,	p, ol, ul, pre {
		margin-top: 0;
	}

	h2, h3, h4, h5, h6 {
		font-weight: normal;
		margin-bottom: 0.2em;
	}

	img {
		max-width: 100%;
		max-height: 100%;
	}

	a {
		text-decoration: none;
	}

	a:link {
  	color: #3794FF;
	}
	
	/* mouse over link */
	a:hover {
		text-decoration: underline;
	}
	
	a:focus,
	input:focus,
	select:focus,
	textarea:focus {
		outline: 1px solid -webkit-focus-ring-color;
		outline-offset: -1px;
	}

	p {
		margin-block-start: 1em;
    margin-block-end: 1em;
		margin-bottom: 0.7em;
		text-align: justify;
  	text-justify: inter-word;
	}

	figure {
		display: flex;
  	align-items: flex-start;
		flex-wrap: wrap;
	}
	
	figure img {
		max-width: 100%;
		max-height: 100%;
		display: block;
  	margin-left: auto;
  	margin-right: auto;
		margin-bottom: 10px;
	}

	figure figcaption {
		text-align: center;
		font-style: italic;
		
		flex-basis: 100%;
	}
	
	ul,	ol {
		margin-bottom: 0.7em;
	}

	hr {
		border: 0;
		height: 2px;
		margin: 0;
	}

	h1, h2, h3, h4, h5, h6, pre {
		font-weight:normal;
		color: inherit;
  }

	h1 {
		padding-bottom: 0.3em;
		line-height: 1.2;
		border-bottom: 1px solid grey;
		font-weight: normal;
		font-size: 2em;
	}

	h2 {
		border-bottom: 1px solid grey;
	}

	strong {
  	color: inherit;
	}

	table, tr, th, td {
		border: none;
	}
	
	th {
		text-align: left;
		border-bottom: 1px solid;
		color: inherit;
	}

	th,	td {
		padding: 5px 10px;
	}

	table > tbody > tr + tr > td {
		border-top: 1px solid grey;
	}
	
	blockquote {
		margin: 0 7px 0 5px;
		padding: 0 16px 0 10px;
		border-left-width: 5px;
		border-left-style: solid;
	}

	code {
		font-family: var(--vscode-editor-font-family, "SF Mono", Monaco, Menlo, Consolas, "Ubuntu Mono", "Liberation Mono", "DejaVu Sans Mono", "Courier New", monospace);
		// font-size: 1em;
		// line-height: 1.357em;
	}
}
```

**Comment by shd101wyy** [comment]
Please note the one defined from `Markdown Preview Enhanced: Customize Css (Workspace)` will override the one from `Markdown Preview Enhanced: Custmoize Css (Global)`

**Comment by josephgarnier** [comment]
I've figured out what happened, but I don't know if it's a bug or not.
After the update, I created an HTML Head and CSS in the Workspace to test, which had the effect of overwriting the global CSS. I then deleted the `.cross` folder once the test was complete. However, the extension didn't detect that the Workspace CSS no longer existed and I had to restart VS Code for it to apply the global CSS.

**Comment by shd101wyy** [comment]
Yes, it requires restart. I will mark it as a bug. Thanks

**Comment by shd101wyy** [comment]
Maybe we should include both the global CSS and workspace CSS :thinking:

**Comment by shd101wyy** [comment]
I just released `0.8.2` which now use both `style.less` from global and workspace.  
Also the bug that you mention from deleting config files but not refreshing the preview should be fixed. I think restarting the vscode is not needed anymore :+1:
